meta-dts-distro/recipes-dts/dts/dts/dts

```diff
@@ -189,57 +140,71 @@ while : ; do
+      echo ${TMP_CLOUDSEND_LOGS_URL} > ${SE_credential_file}
+      echo ${TMP_CLOUDSEND_DOWNLOAD_URL} >> ${SE_credential_file}
+      echo ${TMP_CLOUDSEND_PASSWORD} >> ${SE_credential_file}
 
-      tmpLogsURL="https://cloud.3mdeb.com/index.php/s/${CLOUDSEND_LOGS_URL}/authenticate/showShare"
+      print_green "Dasharo SE credentials has been saved"
+      echo "Veryfing Dasharo SE credentials..."
 
-      CLOUD_DOWNLOAD_URL=`sed -n '2p' < ${SE_credential_file} | tr -d '\n'`
-      CLOUD_PASSWORD=`sed -n '3p' < ${SE_credential_file} | tr -d '\n'`
-      USER_DETAILS="$CLOUD_DOWNLOAD_URL:$CLOUD_PASSWORD"
-      CLOUD_REQUEST="X-Requested-With: XMLHttpRequest"
-
-      if check_network_connection; then
-        CHECK_DOWNLOAD_REQUEST_RESPONSE=$( curl -I -s -f -u "$USER_DETAILS" -H "$CLOUD_REQUEST" "$tmpDownloadURL" -o /dev/null -w "%{http_code}")
-        CHECK_LOGS_REQUEST_RESPONSE=$( curl -I -s -f -H "$CLOUD_REQUEST" "$tmpLogsURL" -o /dev/null -w "%{http_code}")
-        if [ ${CHECK_DOWNLOAD_REQUEST_RESPONSE} -eq 200 ] && [ ${CHECK_LOGS_REQUEST_RESPONSE} -eq 200 ]; then
-          echo ""
-          echo "Cloud credentials are saved and ready to use"
-          export SE_IS_LOGGED=true
-          export CLOUDSEND_PASSWORD="${tmpPASSWD}"
-          export CLOUDSEND_LOGS_URL="${tmpUPURL}"
-          export CLOUDSEND_DOWNLOAD_URL="${tmpDOWNLOADURL}"
-        else
-          echo ""
-          echo -e "Something may be wrong with credentials. Please use option 4 to change SE keys
-                    \rand make sure that there is no typo."
-          rm ${SE_credential_file}
-        fi
+      check_se_creds
+      if [ $? -eq 0 ]; then
+        print_green "Dasharo SE credentials are correct and ready to use"
+      else
+        echo -e "Something may be wrong with credentials. Please use option 4 to change SE keys
+                  \rand make sure that there is no typo."
+        rm ${SE_credential_file}
+        export CLOUDSEND_LOGS_URL="$BASE_CLOUDSEND_LOGS_URL"
+        export CLOUDSEND_PASSWORD="$BASE_CLOUDSEND_PASSWORD"
+        unset CLOUDSEND_DOWNLOAD_URL
+        unset SE_IS_LOGGED
       fi
       ;;
     5)
-      echo "There is no Dasharo firmware update available now for your platform"
-      read -r -p "Please, type any key to back to main menu " OPTION
-      echo
+      if check_if_dasharo; then
+        # For NovaCustom TGL laptops with Dasharo version lower than 1.3.0,
+        # we shall run the ec_transition script instead. See:
+        # https://docs.dasharo.com/variants/novacustom_nv4x_tgl/releases/#v130-2022-10-18
+        if [ "$BOARD_VENDOR" = "Notebook" ]; then
+            case "$SYSTEM_MODEL" in
+              "NS50_70MU"|"NV4XMB,ME,MZ")
+                compare_versions $DASHARO_VERSION 1.3.0
+                if [ $? -eq 1 ]; then
+                # For Dasharo version lesser than 1.3.0
+                  print_warning "Detected NovaCustom hardware with version < 1.3.0"
```

Comment:
@macpijan is this (EC transition by running update, poweroff and then run update once again) a correct process in case of NovaCustom versions < 1.3.0? If yes, then in DTS documentation I would like to add following change https://github.com/Dasharo/docs/pull/541/files#diff-8cfee4c026a23b6eea51be5305d2c9db7d12139283fcec2ee3522ffffc0dc026R462

Reply:
Docuemntation can simply contain "use DTS" as here: https://github.com/Dasharo/docs/pull/545

Reply:
It does say that user should just run update firmware command. Could you please then approve this PR?